Graphs/Table/index.js

```diff
@@ -331,6 +337,41 @@ const TableGraph = (props) => {
         );
     }
 
+    const hoverContent = (cellData, columnIndex, rowIndex) => {
+        return <div key={`tooltip_${cellData}_${columnIndex}_${rowIndex}`}>
+            {cellData} &nbsp;
+        <CopyToClipboard text={cellData ? cellData.toString() : ''}>
+                <button style={{ background: '#000', padding: 1 }} title="copy">
+                    <FaRegClipboard size={10} color="#fff" />
+                </button>
+            </CopyToClipboard>
+        </div>
+    }
+
+    const renderTooltip = (cellData, columnIndex, rowIndex) => {
+        const place = rowIndex === currentStartIndex ? 'bottom' : 'top';
+        const top = rowIndex === currentStartIndex ? (rowHeight * (rowIndex + 1)) - 20 : (rowHeight * rowIndex) - 15;
```

Comment:
Can you please put a code comment on why you're deduction `20` and `15`? Those numbers seem arbitrary without any comment.